### ISSUE TITLE
space:format() call without parameters does not work in a transaction

### ISSUE THREAD

**Issue by bofm** [tarantool/tarantool#3528]
### The error

```
tarantool> do box.begin(); local f = box.space.x:format(); box.space.x:put{1} box.commit() end
---
- error: A multi-statement transaction can not use multiple storage engines
...
```
Please note that the error message is wrong. This is a memtx-only instance.

### How it should work

1. `<some_space>:format()` call without parameters should not throw an error when called in a transaction.
1. `<some_space>:format()` call without parameters should be a lightweight call. This would probably require Tarantool to cache current schema format.

### More info
* Tarantool 1.10.1-135-g193ef4150
* `<space>:format()` call currently uses so-called `sysview` under the hood.

**Comment by kyukhin** [comment]
Issue is that starting from 1-10, to read space format we do (box/lua/schema.lua):
```
-- space format - the metadata about space fields
function box.schema.space.format(id, format)
    local _space = box.space._space
    local _vspace = box.space._vspace
    check_param(id, 'id', 'number')

    if format == nil then
        local tuple = _vspace:get(id)
...
```
So, we do read from `_vspace`, which belong to sysview engine.
That is why presented case fails with `multi-statement` error.

**Comment by locker** [comment]
Fixed by commit 0ecabde8981500886bf5be16cfabe4d1d1a33db4
